services/housetables/src/main/java/com/linkedin/openhouse/housetables/dto/mapper/UserTableVersionMapper.java

```diff
@@ -15,11 +15,21 @@
  */
 @Mapper(componentModel = "spring")
 public class UserTableVersionMapper {
+  public static final String INITIAL_VERSION = "INITIAL_VERSION";
```

Comment:
lets use `com.linkedin.openhouse.common.api.validator.INITIAL_TABLE_VERSION` like used in `OpenHouseInternalRepositoryImpl`

Reply:
ok.